alicloud/data_source_alicloud_rds_instances.go

```diff
@@ -0,0 +1,289 @@
+		Read: dataSourceAlicloudRdsInstancesRead,
+
+		Schema: map[string]*schema.Schema{
+			"name_regex": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"engine": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"db_instance_status": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"db_instance_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"instance_network_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"vpc_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"vswitch_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"connection_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"tags": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"owner_account": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"output_file": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			// Computed values
+			"instances": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"pay_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"region_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"create_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"expire_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"engine": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"engine_version": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_net_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"connection_mode": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"lock_mode": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"lock_reason": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_class": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"instance_network_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"vpc_cloud_instance_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"zone_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"multi_or_single": {
+							Type:     schema.TypeBool,
+							Computed: true,
```

Comment:
I think 'zone_id' can indicate the instance's is multi or single, and there is no need to show multi_or_single. 
In addition, please change 'zone_id' to 'availability_zone' to consistent with other data source.

Reply:
done